src/MapKit/MKMapCameraZoomRange.cs

```diff
@@ -0,0 +1,37 @@
+#if !WATCH // doesn't show up in watch headers
+#if XAMCORE_2_0 || !MONOMAC
+using System;
+using ObjCRuntime;
+
+namespace MapKit {
+
+	public enum  MKMapCameraZoomRange {
+		Min,
+        Max,
+	}
+
+	public partial class MKMapCameraZoomRange {
+		public MKMapCameraZoomRange (double distance) : this (distance, MKMapCameraZoomRangeType.Min)
+		{
+		}
+
+		public MKMapCameraZoomRange (double distance, MKMapCameraZoomRangeType type)
+		{
+			// two different `init*` would share the same C# signature
+			switch (type) {
+			case MKMapCameraZoomRangeType.Min:
+				Handle = InitWithMinCenterCoordinateDistance (distance);
```

Comment:
Use `InitializeHandle (...);` as this will handle some basic checks, e.g. if the handle is nil (`IntPtr.Zero`) then you'll get an _nice_ exception. Otherwise you can end up with a C# instance that is not valid in ObjC lands.